user/Dockerfile

```diff
@@ -69,17 +69,21 @@ ADD install-base-notebook.bash /usr/local/sbin/install-base-notebook.bash
 RUN /usr/local/sbin/install-base-notebook.bash
 
 ADD install-python-packages.bash /usr/local/sbin/install-python-packages.bash
-ADD connectors /usr/local/sbin/connectors-setup
 RUN /usr/local/sbin/install-python-packages.bash
+# Execute all connector setup scripts
+ADD connectors /usr/local/sbin/connectors-setup
+RUN /bin/bash -c 'ls /usr/local/sbin/connectors-setup/*.bash | xargs -L1 /bin/bash'
+
 
 # Since we put NLTK data in /opt rather than /usr/local,
 # we have to explicitly set this
 ENV NLTK_DATA ${CONDA_DIR}/nltk
 
+USER root
```

Comment:
In that case can you move that one command elsewhere and run it as root? This will mix permissions of installed packages and extensions between jovyan and root in /opt/conda, which will cause long term ass-biting to occur.

Also, what is the original source of this binary?